dbp/connect.js

```diff
@@ -0,0 +1,17 @@
+import Sequelize from 'sequelize';
+
+const sequelize = new Sequelize('postgres://postgres:postgres@postgres:5432/postgres');
+
+export const connectToPostgresDb = () => {
+  sequelize
```

Comment:
why not async/await here as well?

Reply:
Good point. Because of copy-paste :) Changed.